guacamole-common/src/main/java/org/apache/guacamole/protocol/GuacamoleClientInformation.java

```diff
@@ -144,5 +149,25 @@ public void setOptimalResolution(int optimalResolution) {
     public List<String> getImageMimetypes() {
         return imageMimetypes;
     }
+    
+    /**
+     * Return the timezone as reported by the client.
+     * 
+     * @return
+     *     A string value of the timezone reported by the client.
```

Comment:
If `null` and/or `""` have special meaning here, that meaning should be documented for the sake of downstream users of this class. I would also suggest settling on a single, specific representation for "no timezone" (presumably `null`?).